centreon/src/Core/HostSeverity/Infrastructure/Repository/DbReadHostSeverityRepository.php

```diff
@@ -164,8 +162,14 @@ public function existsByAccessGroups(int $hostSeverityId, array $accessGroups):
             return $this->exists($hostSeverityId);
         }
 
+        $bindValuesArray = [];
+        foreach ($accessGroupIds as $index => $accessGroupId) {
+            $bindValuesArray[':access_group_id_' . $index] = $accessGroupId;
+        }
+        $bindParamsAsString = implode(',', array_keys($bindValuesArray));
+
```

Comment:
You could make your life way easier by using the `SqlMultipleBindTrait` which allows you then to have something like this

`[$bindValues, $bindQuery] = $this->createMultipleBindQuery($accessGroupIds, ':access_group_id');`

Reply:
Done